dev/kyuubi-extension-spark-3-1/src/main/scala/org/apache/kyuubi/sql/sqlclassification/KyuubiGetSqlClassification.scala

```diff
@@ -28,22 +28,29 @@ import org.apache.kyuubi.sql.KyuubiSQLConf._
  * This object is used for getting sql_classification by the logicalPlan's simpleName.
  * When the configuration item: SQL_CLASSIFICATION_ENABLED is on,
  * we will load the rule from sql-classification-default.json.
+ *
+ * Notice:
+ *  We support the user use the self-defined matching rule: sql-classification.json.
+ *  If there have no this named jsonFile,
+ *  the service will upload the default matching rule: sql-classification-default.json.
  */
 object KyuubiGetSqlClassification {
 
   private val jsonNode: Option[JsonNode] = {
     SQLConf.get.getConf(SQL_CLASSIFICATION_ENABLED) match {
```

Comment:
not related this PR, we don't need check this config here since we check it at use side. That saied we can always load the file but use it depend on the config. can you send a new PR to do this ?